packages/agoric-cli/src/commands/inter.js

```diff
@@ -220,71 +225,17 @@ export const makeInterCommand = (
       `${JSON.stringify(info, bigintReplacer, indent ? 2 : undefined)}\n`,
     );
 
-  const rpcTools = async () => {
-    // XXX pass fetch to getNetworkConfig() explicitly
-    const networkConfig = await getNetworkConfig(env);
-    const { agoricNames, fromBoard, readLatestHead, vstorage } =
-      await makeRpcUtils({ fetch }, networkConfig).catch(err => {
-        throw new CommanderError(1, 'RPC_FAIL', err.message);
-      });
-
-    /**
-     * @param {string} from
-     * @param {number|string} [minHeight]
-     */
-    const storedWalletState = async (from, minHeight = undefined) => {
-      const m = boardSlottingMarshaller(fromBoard.convertSlotToVal);
-
-      const history = await vstorage.readFully(
-        `published.wallet.${from}`,
-        minHeight,
-      );
-
-      /** @type {{ Invitation: Brand<'set'> }} */
-      // @ts-expect-error XXX how to narrow AssetKind to set?
-      const { Invitation } = agoricNames.brand;
-      const coalescer = makeWalletStateCoalescer(Invitation);
-      // update with oldest first
-      for (const txt of history.reverse()) {
-        const { body, slots } = JSON.parse(txt);
-        const record = m.unserialize({ body, slots });
-        coalescer.update(record);
-      }
-      const coalesced = coalescer.state;
-      harden(coalesced);
-      return coalesced;
-    };
-
-    /**
-     * Get OfferStatus by id, polling until available.
-     *
-     * @param {string} from
-     * @param {string|number} id
-     * @param {number|string} minHeight
-     */
-    const pollOffer = async (from, id, minHeight) => {
-      const lookup = async () => {
-        // eslint-disable-next-line @jessie.js/no-nested-await, no-await-in-loop
-        const { offerStatuses } = await storedWalletState(from, minHeight);
-        const offerStatus = [...offerStatuses.values()].find(s => s.id === id);
-        if (!offerStatus) throw Error('retry');
-        harden(offerStatus);
-        return offerStatus;
-      };
-      const retryMessage = 'offer not in wallet at block';
-      const opts = { ...networkConfig, execFileSync, delay, retryMessage };
-      return pollBlocks(opts)(lookup);
-    };
-
-    return {
-      networkConfig,
-      agoricNames,
-      fromBoard,
-      vstorage,
-      readLatestHead,
-      storedWalletState,
-      pollOffer,
-    };
+  const tryMakeUtils = async () => {
```

Comment:
like any promise it may reject, so shouldn't be qualified with `try`. It shouldn't be named `makeWalletUtils` but it could be,
```suggestion
  const makeEnvWalletUtils = async () => {
```
or something else concise to convey that it's going to read env or work only in this module

Reply:
The main thing that it's supposed to do is treat RPC errors as reported errors (with `CommanderError`) rather than a crash. That's what I meant by "try". I'm struggling to find a more clear name.